cdap-docs/reference-manual/source/http-restful-api/metrics.rst

```diff
@@ -68,33 +74,33 @@ enclosing context. These are the available Application contexts of CDAP:
    * - System Metric
      - Context
    * - One Flowlet of a Flow
-     - ``namespace.<namespace>.app.<app-id>.flow.<flow-id>.flowlet.<flowlet-id>``
+     - ``namespace:<namespace> app:<app-id> flow:<flow-id> flowlet:<flowlet-id>``
```

Comment:
should include run-id here, otherwise we aggregate flowlet of a flow between runs, same applies for examples below that uses program types.